nav-authsql/_run.detached.bat

```diff
@@ -1,7 +1,7 @@
 set /p host=<__privatereponame.txt
 set /p ImageName=<__imagename.txt
-set ContainerName=%ImageName:/=_%
+set ContainerName=%ImageName::=_%
```

Comment:
I can`t understand this changes, sorry. In this cases (this example of NAV 2013 with CU3) I want to achieve image "navservice" with the version "2017.cu3", so in the docker words the following iamge:tag => "navservice:2017.cu3".
Your changes lead to "navservice_2017.cu3" so I don`t understand well what was your intention. Maybe my mistake... That`s why asking.